app/src/main/java/com/heapixLearn/discovery/logic/contact/LogicContactManager.java

```diff
@@ -0,0 +1,103 @@
+package com.heapixLearn.discovery.logic.contact;
+
+import android.os.Handler;
+import android.os.Looper;
+
+import com.heapixLearn.discovery.logic.contact.contacts_domain_interfaces.DBStoreContactManager;
+import com.heapixLearn.discovery.logic.contact.contacts_domain_interfaces.ServerStoreContactManager;
```

Comment:
@dianaprimshits do not combine camel case and underscore case at the same time, we agreed to use camel case

Reply:
@YodaMike Maybe you did't hear. We have agreed to use underscores case in the package names and camelcase in the class names.